modules/ingest-geoip/build.gradle

```diff
@@ -22,12 +22,12 @@ esplugin {
 }
 
 dependencies {
-  api('com.maxmind.geoip2:geoip2:2.13.1')
+  api('com.maxmind.geoip2:geoip2:3.0.0')
   // geoip2 dependencies:
   api("com.fasterxml.jackson.core:jackson-annotations:${versions.jackson}")
```

Comment:
Jackson is no longer bound to the server version, so this can be upgraded as needed. I see that geoip2 uses Jackson 2.13.1.

Reply:
Also note, if Jackson is not used by our own geoip code, then it can be switched to a runtime dependency (this would match what it would be if we had transitive dependencies turned on).